bucket/onlyoffice.json

```
@@ -0,0 +1,37 @@
+﻿{
+    "version": "5.3.5",
+    "license": "AGPLv3",
+    "homepage": "https://www.onlyoffice.com/apps.aspx",
+    "innosetup": true,
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/ONLYOFFICE/DesktopEditors/releases/download/ONLYOFFICE-DesktopEditors-5.3.5/DesktopEditors_x64.exe#/dl.7z",
```

```suggestion
            "url": "https://github.com/ONLYOFFICE/DesktopEditors/releases/download/ONLYOFFICE-DesktopEditors-5.3.5/DesktopEditors_x64.exe",
```

If it's an innosetup installer, keep exe.